app/src/main/java/com/cornellappdev/transit/ui/screens/RouteScreen.kt

```diff
@@ -0,0 +1,164 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.TextUnitType
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.transit.R
+import com.cornellappdev.transit.ui.components.MenuItem
+import com.cornellappdev.transit.ui.theme.DividerGrey
+import com.cornellappdev.transit.ui.theme.MetadataGrey
+import com.cornellappdev.transit.ui.theme.PrimaryText
+import com.cornellappdev.transit.ui.theme.SecondaryText
+import com.cornellappdev.transit.ui.theme.sfProDisplayFamily
+import com.cornellappdev.transit.ui.viewmodels.HomeViewModel
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+
+@OptIn(ExperimentalPermissionsApi::class, ExperimentalMaterial3Api::class)
+@Composable
+fun RouteScreen(
+    //homeViewModel: HomeViewModel = hiltViewModel()
+) {
+
+    Column(modifier = Modifier.fillMaxSize()) {
+        //TODO make an AppBarColors class w/ the right colors and correct icon
+        TopAppBar(
+            title = { Text(text = "Route Options", fontFamily = sfProDisplayFamily, fontStyle = FontStyle.Normal)},
+            navigationIcon = {
+                IconButton(onClick = { /*TODO make this button nav back to whatever screen*/ }) {
+                    Icon(
+                        imageVector = Icons.Outlined.KeyboardArrowLeft,
+                        contentDescription = ""
+                    )
+                }
+            }
+
+        )
+
+        Divider(thickness = 1.dp, color = DividerGrey)
+        Row(
+            modifier = Modifier
+                .padding(horizontal = 20.dp, vertical = 12.dp),
+            horizontalArrangement = Arrangement.spacedBy(16.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            // From-to column
+            Column(verticalArrangement = Arrangement.spacedBy(24.dp)) {
+                Text(
+                    text = "From",
+                    color = PrimaryText,
+                    fontFamily = sfProDisplayFamily,
+                    fontWeight = FontWeight.Normal
+                )
+                Text(
+                    text = "To",
+                    color = PrimaryText,
+                    fontFamily = sfProDisplayFamily,
+                    fontWeight = FontWeight.Normal
+                )
+            }
+
+            Icon(
+                painter = painterResource(id = R.drawable.route_icon),
+                contentDescription = "",
+                modifier = Modifier.size(width = 16.dp, height = 54.dp),
+                tint = Color.Unspecified
+            )
```

Comment:
Oookay so this is a bit of a sticky situation because the icon doesn't really line up perfectly with the text. I'd recommend instead exporting the top bubble and bottom bubble as separate `SVG`s (not pngs, see my note below!) and making the line manually, so that we can make it line up.

Reply:
fixed with ConstraintLayout (talked about it @ informal work session)

![image](https://github.com/cuappdev/ithaca-transit-compose/assets/63558657/7a5b18bd-5071-406c-8ca0-c314bb3fbd7d)